index.html

```diff
@@ -710,50 +710,33 @@ <h4>Proof Configuration (eddsa-rdfc-2022)</h4>
 identifier (<var>cryptosuite</var>). A <em>proof configuration</em>
 object is produced as output.
           </p>
-
           <ol class="algorithm">
             <li>
-Let <var>proofConfig</var> be an empty object.
-            </li>
-            <li>
-Set <var>proofConfig</var>.<var>type</var> to
-<var>options</var>.<var>type</var>.
+Let |proofConfig| be a clone of the |options| object.
             </li>
             <li>
-If <var>options</var>.<var>cryptosuite</var> is set, set
-<var>proofConfig</var>.<var>cryptosuite</var> to its value.
-            </li>
-            <li>
-If <var>options</var>.<var>type</var> is not set to `DataIntegrityProof` and
-<var>proofConfig</var>.<var>cryptosuite</var> is not set to `eddsa-rdfc-2022`,
-an error MUST be raised and SHOULD convey an error type of
+If |proofConfig|.|type| is not set to `DataIntegrityProof` and/or
+|proofConfig|.|cryptosuite| is not set to `eddsa-rdfc-2022`, an
+error MUST be raised and SHOULD convey an error type of
 <a data-cite="VC-DATA-INTEGRITY#PROOF_GENERATION_ERROR">PROOF_GENERATION_ERROR</a>.
             </li>
             <li>
-Set <var>proofConfig</var>.<var>created</var> to
-<var>options</var>.<var>created</var>. If the value is not a valid
-[[XMLSCHEMA11-2]] datetime,
-an error MUST be raised and SHOULD convey an error type of
+If |proofConfig|.|created| is set to a value that is not a
+valid [[XMLSCHEMA11-2]] datetime, an error MUST be
```

Comment:
So `proofConfig.created` is optional and this doesn't read that way. If we have similar text in other cryptosuites we'll need to fix them as well. Maybe this works:

```suggestion
If |proofConfig|.|created| is present and set to a value that is not a
valid [[XMLSCHEMA11-2]] datetime, an error MUST be
```